src/reducers/index.js

```diff
@@ -17,6 +18,7 @@ const sessionPersistConfig = {
 const rootReducer = history =>
   combineReducers({
     form,
+    openform,
```

Comment:
Yeah, I wanted to name it form, but is taken by redux-form, will try to change the name of redux form reducer, and see what happens, otherwise, I will have to keep this name until I'm done removing redux-form